packages/shared/styles/styleUtils/typography/color.ts

```diff
@@ -6,9 +6,6 @@ import {
   themeTextColorSecondaryInverted
 } from "../../../../design-tokens/build/js/designTokens";
 
-// TODO: when we do the design tokens lib,
-// generate a type for _only_ our colors
-// instead of just accepting any string
 export const tintText = (color: React.CSSProperties["color"]) => css`
```

Comment:
I don't have the full proper context to see the need for this one, and it is 5 years old. I feel we can safely remove.